main/zoe/api/Zoe-Intro.md

```diff
@@ -0,0 +1,109 @@
+- *terms*: Another set of key : value pairs.
+  -   These customize contract aspects expressed in the contract without specific values. For example, an auction contract’s may not define what the minimum bid has to be or the minimum amount one has to raise from the current high bid, just that there are such values which the contract uses. These values are different for a collection of used Harry Potter paperbacks and a signed first British hardback edition of *Harry Potter and the Philosopher’s Stone*. If you write code in the contract to access that instance’s `terms`’ name:value pairs of `('MinBid' : 25, 'Currency : 'USDollar', 'MinRaise' : 5)` or  `('MinBid' : 100000, 'Currency : 'USDollar', 'MinRaise' : 5000)`, you’ve customized the generic contract for that specific instance. 
+
+
+`zoe.makeInstance()` returns an *invite* to be a party in that contract instance. Thus, the party who made the contract instance has an invite to it. The `instanceHandle`, the unique identifier for the contract instance, can be retrieved from the invite using `GetInstanceHandle()`.
+
+
+In order to get their *seat* at the contract instance, its creator must *redeem* their invite to it. 
+This redemption is also where the creator make their *offer* in the form of a *proposal*. Zoe destroys invites on their redemption, so they can’t be used by anyone else. Even if they have a reference to the invite object. More details on redemption, offers, and proposals below.
+
+
+Using their seat at the contract instance, any user can create new invites to the offer and its contract instance. Once a contract party holds an invite to an offer, they can send it to anyone they'd like to also possibly be a party to the contract. In other words, usually someone who has the asset you want to get. 
+
+
+Finding these other parties to invite is out of Zoe or the contract’s scope in most cases (you could possibly write something into the contract that seeks out potential partners, but that’s way beyond this introduction’s purpose). You have to either already know of potential other parties, for example, from past experience trading with them, or communicate somewhere outside the contract to find them.
+
+
+*Invite*s are ERTP *payments*, so the same principles apply to them as to what you usually think of ERTP payments. You can send the same payment or invite to multiple parties...but the first party to *redeem* it removes the value from the shared payment. As part of redeeming it, they then create their own new payment object with the same amount. Remember, Zoe destroys invites on their redemption.
```

Comment:
>  As part of redeeming it, they then create their own new payment object with the same amount.

I think this is confusing ERTP's `claim' method, which is what (under the hood) creates a new payment for a user containing the amount  yanked from the previous (possibly shared) payment. Redeeming `burns` the payment, which means that the value - the amount minted that is stored in the payment - is deleted.